bin/functions.d/eXist-settings.sh

```diff
@@ -76,7 +76,7 @@ restore_library_path() {
 
 set_client_java_options() {
     if [ -z "${CLIENT_JAVA_OPTIONS}" ]; then
-	CLIENT_JAVA_OPTIONS="-Xms128m -Xmx1024m -Dfile.encoding=UTF-8";
+	CLIENT_JAVA_OPTIONS="-Xms128m -Xmx2048m -Dfile.encoding=UTF-8";
```

Comment:
If you run backup/restore via the client having a larger max memory is sometimes required. For normal operations, the memory won't be used, so setting it a bit higher won't do harm I think?